bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -59,85 +51,66 @@
   ) %}
 
   <h2 class="c-primary-cta-logo"><img alt="Firefox" src="{{ static('protocol/img/logos/firefox/logo-word-hor.svg') }}" height="40" width="125"></h2>
-  <h3 class="c-primary-cta-title">{{ _('Fast for good.') }}</h3>
+  <h3 class="c-primary-cta-title">{{ ftl('home-fast-for-good') }}</h3>
 
   <p class="c-primary-cta-desc">
-    {{ _('With 2x the speed, built-in privacy protection and Mozilla behind it, the new Firefox is the better way to
-    browse.') }}
+    {{ ftl('home-with-2x-the-speed-built-in') }}
   </p>
 
   {{ download_firefox_thanks(dom_id='download-intro', download_location='primary cta') }}
   {% endcall %}
 
   <div class="mzp-l-content mzp-t-mozilla">
     {{ billboard(
-    title=_('We make the internet safer, healthier and faster for good.'),
-    desc=_('Mozilla is the not-for-profit behind Firefox, the original alternative
-    browser. We create products and policy to keep the internet in service
-    of people, not profit.'),
+    title=ftl('home-we-make-the-internet-safer'),
+    desc=ftl('home-mozilla-is-the-not-for-profit'),
     image_url='img/home/dino.svg'
     )}}
 
     <section class="c-column-container">
       <div class="c-column">
         <div class="c-column-content">
-          <h2>{{ _('Our impact') }}</h2>
+          <h2>{{ ftl('home-our-impact') }}</h2>
           <p>
             {% if l10n_has_tag('home-update-20171116') %}
```

Comment:
Does `l10n_has_tag` still work with Fluent? I think this should be `ftl_has_messages`. 

BUT given how old these strings are maybe we don't need any conditional here and just go with the latest string and delete the other two.

Reply:
Good catch, will update 👍